### ISSUE TITLE
don't recommend functions that use 32-bit date and case indexes

### ISSUE THREAD

**Issue by jeffkap** [qomaio/seamless-r-examples#1]
For background information - please refer to this page in the FAME documentation: https://fame.sungard.com/support_secure/fame/online_help_115/chli/replacement_C_HLI_functions.htm

As a general rule, the "old chli functions"  in the left hand column should never be used in demos.  Most of those old functions have 32-bit integer arguments representing case numbers or dates.  We should always steer people to the current functions which have 64-bit integer arguments representing dates instead.

Obsolete functions which *don't* have date arguments are:

cfmsinp - the replacement function returns a longer string
cfmsatt / cfmgtatt - the replacement functions protect against using the wrong type
cfminwc / cfmnxwc / cfmgnam - the replacement functions can return long names
cfmufrq / cfmpfrq - the replacement functions are more user-friendly 

-----

The specific issue I noticed is that FameR/chartscodeRHLI.html talks about cfmsrng, cfmpdat and cfmdatp which are "old functions".  Please change those over to the corresponding new functions that take 64-bit date arguments.

**Comment by krkeane** [comment]
@jeffkap  Thanks for the feedback!

As we discussed, I don't like having two calling patterns, ```void cfm*()``` and ```int fame_*()```. What do you think about us macro'n our way to consistency?

```
#define fame_initialize() ({int status; cfmini(&status); status;})
```

Then, a user can live in the void ```cfm*()``` world or the ```int fame_*()``` world.

**Comment by jeffkap** [comment]
Yes, I like this idea a lot!  Why not?  How much dev and test effort would it take?  I suppose you don't have to write all of the macros up front.  Just write a few at a time as needed.
